config/settings.yml

```diff
@@ -513,11 +476,9 @@
       :name: Import/Export
     :migration:
       :critical:
-      - DatastoreFileMovedEvent
       - DrsMigrateVM_Task
       - DrsVmMigratedEvent
```

Comment:
This is a valid VMware Event https://www.vmware.com/support/developer/vc-sdk/visdk400pubs/ReferenceGuide/vim.event.DrsVmMigratedEvent.html